duckduckgo/Android
# Feature/remove url phishing possibility

Task/Issue URL: https://geekboy.ninja/p0c/spoof-bar.html
Tech Design URL: 
CC: 

**Description**:
Removes the ability for a malicious website to perform a specific type of phishing attack

**Steps to test this PR**:
1. Visit https://geekboy.ninja/p0c/spoof-bar.html
1. Note that the URL bar **does not** show gmail while it is attempting to redirect and load another webpage
1. Note that the page will eventually timeout and show an error in the `WebView`. At this point, it will update the URL bar to show the URL which didn't load (gmail)
1. Ensure the forward/back buttons continue to work as expected, as they have been updated as part of this work


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
